packages/compartment-mapper/src/types.js

```diff
@@ -214,20 +215,6 @@ export {};
  * @returns {ImportHook}
  */
 
-/**
- * @typedef {object} SourceMapHookDetails
- * @property {string} compartment
- * @property {string} module
- * @property {string} location
- * @property {string} sha512
- */
-
-/**
- * @callback SourceMapHook
- * @param {string} sourceMap
- * @param {SourceMapHookDetails} details
- */
-
```

Comment:
Deduplicating this interface implies a tighter coupling than necessarily exists.

Reply:
Motivation was to fix a type error in their usage. The one kept fits the code in use.

Do you prefer duplicating that one instead of importing it?

Reply:
Went with duplicating. PTAAL.